.chloggen/update-attribute-sqlserverreceiver-part-1.yaml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sqlserverreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: |
+   adde a new metric resource attribute `host.name`
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [40141]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: | 
+  - add `host.name` resource attributes in the exported metrics. in document it says this attribute is exported but actually not.
```

Comment:
I don't understand what this is supposed to mean. After checking, I believe you're saying that before this PR it was documented that the resource attribute `host.name` was being exported, but in reality it was not. This PR resolves this by adding the functionality to export this resource attribute. Is that correct?

If my comment is correct, can you clarify this comment a bit?